### ISSUE TITLE
Are all these papers worth reading?

### ISSUE THREAD

**Issue by zhufeng888** [timzhang642/3D-Machine-Learning#65]
I'm a green hand on 3D point cloud recognition, and I'm finding some learning materials like this. But there are so many papers and I have a question about the principle that these papers are recorded. 
Do these papers represent milestones and they are all ecellent compared to most other normal papers，or are these papers just collected in terms of their context without any standard ?
Because there are so much knowledge on 3D machine-learing that it consumes too much of my energy, I want to find what is more important for beginners, and I think many beginners  also have this problem. It will be great for me if some papers are recommend in different levels，such as from one star to five stars. 
Finally, I really appreciate your efforts.

**Comment by zhufeng888** [comment]
For example, if you think the paper _PointNet_   represents  a milestone in the field which is valuable for others to learn from, you will label it with five stars ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ like your cutting line.

**Comment by erinaldi** [comment]
You are right, there are a lot of resources. As a beginner, it is worth focusing on survey papers where you can get an overview of the field and of the milestone papers/studies. 

Then you can move to other resources like [this](https://github.com/Yochengliu/awesome-point-cloud-analysis) or read specific papers following the survey ones.

**Comment by zhufeng888** [comment]
It's so fast to get the help from erinaldi,and the link you provide is very valuable to me. Thank you very much.

**Comment by nitinagarwal** [comment]
I recently gave a [3hr talk](https://tinyurl.com/wvmtv76) on 3D deep learning in the graduate class (Adv. Computer Graphics) @ UCI, where I provided an overview of the area + a lot of resources useful for anyone doing research in this direction. Hope you find it useful.

**Comment by zhufeng888** [comment]
Thank you nitinagarwal, your notes are great to help me have a holistic understanding especially these pictures that add additional extension information, making it easier to understand.
